pwa/src/templates/schemasTemplate/SchemasTemplate.tsx

```diff
@@ -90,20 +90,22 @@ export const SchemasTemplate: React.FC = () => {
 
                   <TableCell>{translateDate(i18n.language, schema.dateModified)}</TableCell>
 
-                  <TableCell className={styles.callsToAction}>
-                    <Button
-                      onClick={(e) => goToCreateObject(e, schema.id)}
-                      variant="primary"
-                      icon={faPlus}
-                      label={t("Add Object")}
+                  <TableCell>
+                    <ActionButton
+                      actions={[
+                        {
+                          type: "add",
+                          onSubmit: () => goToCreateObject(schema.id),
+                          label: "add Object",
```

Comment:
I'd pass this like `"Add Object"`, regardless of it being `_.upperFirst` in its implementation.